agent-flow/agent_flow/workflows/perf_optimize/README.md

```
@@ -3,28 +3,26 @@
-                 false)         │  benefit;    │  pending item)  REJECT | PUSH_BACK;││
-                                │  profiles    │                 ≤ max_attempts     ││
-                                │  after an    │                 _per_item retries) ││
-                                │  accept,     └────────────────────────────────────┘│
-                                │  else re-plans, no GPU)                             │
-                                └─────────────────────────────────────────────────────┘
-
---reuse-analysis <dir> imports a previous perf-analyze / perf-optimize
+`--reuse-analysis <dir>` imports a previous perf-analyze / perf-optimize
 run's baseline, SOL projection and profile, starting the campaign at the
 optimize stage (round 1's analyzer then plans without profiling).
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)